orc8r/gateway/python/magma/magmad/metrics_collector.py

```diff
@@ -27,6 +30,18 @@
 from magma.common.service_registry import ServiceRegistry
 
 
+class ScrapeTarget(object):
```

Comment:
use `typing.NamedTuple` for data classes: https://docs.python.org/3.5/library/typing.html